lua/autorun/server/sv_restrict_propcore.lua

```diff
@@ -59,9 +55,10 @@ local function adminOnlyCondition( self, ... )
 end
 
 local function restrictedCondition( self, ... )
-    if self.player:IsAdmin() then return true end
+    local canUse, message = hook.Run( "CFC_RestrictPropCore_CanUse", self.player, self )
+    if canUse ~= nil then return canUse, message end
```

Comment:
Means none of the other conditions are checked if the hook returns `true`, is this intentional?

Reply:
Yep!